lib/update-fetch.js

```diff
@@ -127,7 +127,7 @@ exportables.fetchBuild = function(build) {
 
 exportables.downloadTgz = function(tgzUrl, fileMap) {
   return new Promise((resolve, reject) => {
-    log.info('Downloading files. This may take a few minutes...');
+    log.info('Downloading files (this may take a few minutes)...');
```

Comment:
Do we still need "(this may take a few minutes)" here if it was removed in the other file?